clustered/client/src/main/resources/ehcache-clustered-ext.xsd

```diff
@@ -57,7 +57,23 @@
         <xs:annotation>
           <xs:documentation>
             Specifies the amount of time a cache read operation will wait for a response from a cluster
-            server before abandoning the cluster operation and returning a null.
+            server before abandoning the cluster operation.
+          </xs:documentation>
+        </xs:annotation>
+      </xs:element>
+      <xs:element name="mutative-timeout" type="eh:time-type" minOccurs="0" maxOccurs="1">
+        <xs:annotation>
+          <xs:documentation>
+            Specifies the amount of time a cache mutative operation will wait for a response from a cluster
+            server before abandoning the cluster operation.
+          </xs:documentation>
+        </xs:annotation>
+      </xs:element>
+      <xs:element name="lifecycle-timeout" type="eh:time-type" minOccurs="0" maxOccurs="1">
```

Comment:
This is the connection timeout now right?  Also given it's a connection timeout wouldn't it make sense for this to be more tightly associated with the connection related stuff?

```xml
<tc:connection url="terracotta://example.com:9540,example.com:9640/cachemanager">
  <tc:timeout unit="seconds">30</tc:timeout>
</tc:connection>
```

Reply:
No. Was renamed connection-timeout and that's it. Need to be discussed?